Useful tools & utilities/Figma.md

```diff
@@ -0,0 +1,124 @@
+As a developer, understanding the fundamentals of the design tools used within your team is an essential skill. The notorious war between
+developers and designers is never-ending and design tools are the places where these two worlds collide. This raises the importance of using
+good design tools which provide a nice connection between the design and development process. Imagine if all the screen designs, components,
+different assets were stored on Google Drive and the review process and communication was handled through email/Slack. To get a single
+component, you'll have to go through several folders and then, you'll have to open your email/Slack to remind yourself about the discussions
+you had regarding the implementation. This isn't a big deal if you have to do it once in a day, however, this is a continuous and frequent
+process. Eventually, the project grows, new designs are requested, new folders and files are added and all of a sudden, handling designs
+like this sounds like a nightmare, right?
+
+![Designers vs Developers meme](/img/figma/figma_designers_vs_developers_meme.png "Designers vs Developers meme")
+
+[Figma](https://www.figma.com) is a cloud-based design tool which provides simple, effective and intuitive functionalities perfect for team
+collaboration. Similar to Google Docs, team members can easily communicate, collaborate and edit the same file in Figma in real time.
+
+## Benefits of using Figma
```

Comment:
This whole section feels like you are getting a commission from Figma 🤔 :sus:
But we can leave it since this will be available to the general public as well.